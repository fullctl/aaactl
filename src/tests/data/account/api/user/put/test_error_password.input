{
  "username": "foobar",
  "first_name": "foo",
  "last_name": "bar",
  "email": "foobar@localhost",
  "password": "wrong password"
}
