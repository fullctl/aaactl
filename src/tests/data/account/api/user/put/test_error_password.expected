{"data": [], "errors": {"password": ["Incorrect password"]}, "profiling": {"queries": 0}}
